pymc/smc/kernels.py

```diff
@@ -193,10 +193,16 @@ def initialize_population(self) -> dict[str, np.ndarray]:
             )
 
             model = self.model
+
+            # Respect custom initvals (issue #7438)
+            initval_strats = {
+                **model.rvs_to_initial_values,
```

Comment:
Won't this override the default behavior of sampling from the prior, to using the moment?